### PR TITLE
Code Coverage at 100%

### PR DESCRIPTION
**Pull request by WolframHempel**
Sorry for amount of changes, but I've also removed whitespace, unnecessary functions, added semicolons and other bits I found along the way.

### CODE REVIEW

File: test/utils/dependency-initialiserSpec.js [hunk 1]
```diff
@@ -27,11 +26,74 @@ describe( 'dependency-initialiser', function(){
...
+	var log = jasmine.createSpy( 'log' );
+	var originalProcessExit = process.exit;
+	var originalConsoleLog = console.log;
+	var options = {
+		plugin: new PluginMock( 'A' ),
+		logger: { log: jasmine.createSpy( 'log' ), isReady: true },
+		dependencyInitialisationTimeout: 1
+	};
+
+	it( 'disables process exit', function(){
```
Comment (yasserf) [inline test/utils/dependency-initialiserSpec.js]: these should be beforeAll and afterAll since they don't actually validate anything. Just saying =D

Comment (WolframHempel) [inline test/utils/dependency-initialiserSpec.js]: They used to be - it breaks (try it if you like). Something is different in their timing, it is called before process.exit is called,

Comment (yasserf) [inline test/utils/dependency-initialiserSpec.js]: Interesting.

**Comment by WolframHempel** [comment]
@adityamenon Good spot, but our problem actually is that we run our build with multiple versions of node, all the way back to 0.12 - which doesn't support ES6 syntax yet.

At some point we will abandon support for Node versions <1, but I feel it needs to be for a more crucial reason than me being to lazy to type `function` :-)

**Pull request merged**